### PR TITLE
Fix translations of 'Branding' page in Suomi view

### PR DESCRIPTION
**Pull request by manzapanza**
issue: https://github.com/apinf/platform/issues/1955

### CODE REVIEW

File: branding/logo/client/upload/uploadButton/uploadButton.html [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <template name="uploadProjectLogoButton">
-  <a class="btn btn-primary fileBrowse">Upload Logo</a>
+  <a class="btn btn-primary fileBrowse">{{_ "uploadProjectLogoButton_uploadButton_uploadLogo" }}</a>
```
Comment (brylie) [inline branding/logo/client/upload/uploadButton/uploadButton.html]: To make this line shorter, put the i18n token on its own line:

```html
<a class="btn btn-primary fileBrowse">
    {{_ "uploadProjectLogoButton_uploadButton_uploadLogo" }}
</a>
```

**Comment by brylie** [comment]
Add the 'ready for review' label when this PR is ready for review.

File: branding/collection/schema.js [hunk 2]
```diff
@@ -48,14 +48,14 @@ Branding.schema = new SimpleSchema({
     type: String,
     optional: true,
     allowedValues: ['Facebook', 'Twitter', 'Github'],
+    autoform: {
+      firstOption: TAPi18n.__('schemas.branding.socialMedia.$.name.firstOption'),
```
Comment (manzapanza) [inline branding/collection/schema.js]: @brylie I don't know why, but theoretically, I should set a firstOption label without this schema autoform configuration. As I did for the other field 'socialMedia.$.url'.

Comment (brylie) [inline branding/collection/schema.js]: Does this not work as expected? What is the result?

Comment (brylie) [inline branding/collection/schema.js]: Try without the `.$`

**Pull request merged**